### PR TITLE
Add UniDAO

### PR DESCRIPTION
**Pull request by densmirnov**

### CODE REVIEW

File: src/namespaces.json [hunk 1]
```diff
@@ -198,5 +198,24 @@
     "min": 0,
     "invalid": [],
     "visible": false
+  },
+  "unidao": {
+    "key": "unidao",
+    "symbol": "MRQ",
+    "name": "UniDAO",
+    "defaultView": "all",
+    "address": "0xc1d204fc521f9e125a55dd8b2152d7344766cc21",
```
Comment (bonustrack) [inline src/namespaces.json:207]: The address need to be checksumed
```suggestion
    "address": "0xC1d204fC521F9e125a55dd8b2152d7344766cC21",
```

File: src/namespaces.json [hunk 2]
```diff
@@ -198,5 +198,24 @@
...
+    "token": "0xc1d204fc521f9e125a55dd8b2152d7344766cc21",
```
Comment (bonustrack) [inline src/namespaces.json:208]:
```suggestion
    "token": "0xC1d204fC521F9e125a55dd8b2152d7344766cC21",
```

File: src/namespaces.json [hunk 3]
```diff
@@ -198,5 +198,24 @@
     "min": 0,
     "invalid": [],
     "visible": false
+  },
+  "unidao": {
+    "key": "unidao",
+    "symbol": "MRQ",
```
Comment (bonustrack) [inline src/namespaces.json:204]: Isn't the symbol and name supposed to be: MarsReputation (MRR)?

Comment (densmirnov) [reply src/namespaces.json:204]: Yes, correct! Thank you a lot!

Comment (densmirnov) [reply src/namespaces.json:204]: MRR is one of the reputation tokens in UniDAO Ecosystem, currently used for governance. So, yes, it's correct name (UniDAO), but incorrect naming,

**Review by bonustrack — APPROVED** [review]
Looking good!

**Pull request merged**